airtime_mvc/application/modules/rest/controllers/MediaController.php

```diff
@@ -200,8 +206,12 @@ public function putAction()
                 ->setHttpResponseCode(200)
                 ->appendBody(json_encode(CcFiles::sanitizeResponse($file)));
         } else if ($file) {
+            // Sanitize any incorrect metadata that slipped past validation
+            FileDataHelper::sanitizeData($whiteList);
+
```

Comment:
Yeah, same comment as above. Rather than putting this boilerplate in 3 spots, it can go in one spot at the top of validateRequestData()